src/FormBuilder/resources/js/form-builder/src/settings/form-summary/index.js

```diff
@@ -17,6 +18,10 @@ const FormSummarySettings = () => {
                     label={__('Title')}
                     value={formTitle}
                     onChange={(formTitle) => dispatch(setFormSettings({formTitle}))}
+                    onBlur={(event) => {
```

Comment:
I wondering if there is a good way to centralizer this sanitation.

@jonwaldstein what if we added a new export method to the form state reducer:

```ts
export function setFormTitle(formTitle) {
    return {
        type: UPDATE_SETTINGS,
        settings: {
                formTitle: stripTags(formTitle)
        }
    };
}
```